Week1/homework/style.css

```diff
@@ -1 +1,53 @@
-/* add your styling here */
\ No newline at end of file
+/* add your styling here */
+*{
+    padding: 0;
+    margin: 0;
+}
+body{
+    background-color: rgb(18, 51, 78);
+}
+h1{
+    text-align: center;
+    font-size: 4rem;
+    color: rgb(0, 135, 253);
+    text-decoration: underline;
+    text-shadow: 1px 1px 5px rgb(6, 6, 46);
+    margin: 2%;
+}
+ul{
+    list-style: none;
+    display: flex;
+    flex-wrap: wrap;
+}
+ul li{
+    box-shadow: 1px 1px 5px black;
```

Comment:
Your page looks pretty nice. Only one thing. On wide screens the images are stretched horizontally:
<img width="900" alt="Screenshot 2019-08-21 at 21 00 56" src="https://user-images.githubusercontent.com/503813/63460439-cbfb8200-c456-11e9-8f1a-159280f32062.png">


Please find a way to solve that.

Reply:
Thank you,
I’ll fix it .